test/TestAsset/http-status-codes.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Specifically, the test should just download them by pinging https://www.iana.org/assignments/http-status-codes/http-status-codes.xml